docs/docs/Components/components-logic.md

```diff
@@ -0,0 +1,159 @@
+# Logic
+
+## Conditional router
+
+This component routes an input message to a corresponding output based on text comparison.
+
+The ConditionalRouterComponent routes messages based on text comparison. It evaluates a condition by comparing two text inputs using a specified operator and routes the message accordingly.
+
+### Parameters
+
+#### Inputs
```

Comment:
You should not have "empty" headings. In other words, a heading with no text. I don't think this heading is needed.